clojure-hacker-news.iml

```diff
@@ -17,10 +17,29 @@
     </content>
```

Comment:
we add `*.iml` files to the `.gitignore` as these files serve IntelliJ specific configuration

Reply:
You can generate pretty wholesome `.gitignore`s using gitignore.io. Just add your editor/s and the languages used. You'll never have to think of those anymore ;)

Reply:
Thanks, removed the iml file in a separate PR.